hydra-node/test/Hydra/BehaviorSpec.hs

```diff
@@ -448,7 +471,12 @@ spec = parallel $ do
               send n1 Close
               waitUntil [n1, n2] $ ReadyToFanout{headId = testHeadId}
               send n1 Fanout
-              waitUntil [n1, n2] $ HeadIsFinalized{headId = testHeadId, utxo = utxoRefs [1, 2]}
+
+              waitMatch n2 $ \case
+                HeadIsContested{headId, snapshotNumber} -> guard $ headId == testHeadId && snapshotNumber == 1
+                _ -> Nothing
```

Comment:
Why did we add this? Why is a contest expected here?

Reply:
One node n2 issues a decommit and  n1 closes. n2 already managed to get the confirmed snapshot with something to decrement so I wanted to assert that contestation happened therefore it is ok to just fanout `utxoRef 1` instead of `utxoRefs [1,2]`